src/main/webapp/index.jsp

```diff
@@ -106,7 +106,7 @@
             <h2 class="media-heading">Not for Profit:</h2>
             <p>
                 TEAMMATES does not have commercial ambitions. It is funded mainly by education grants from the National University of Singapore.
-                TEAMMATES also received funding support under the <a href="https://developers.google.com/open-source/gsoc/" target="_blank"> Google Summer of Code</a> program in 2014 and 2015.
+                TEAMMATES also received funding support under the <a href="https://developers.google.com/open-source/gsoc/" target="_blank"> Google Summer of Code</a> program in 2014,2015 and 2016.
```

Comment:
"2014,2015 and 2016" -> "2014, 2015, and 2016" (missing a whitespace and a comma)